src/probabilit/distributions.py

```diff
@@ -241,7 +269,51 @@ def _pert_to_beta(minimum, mode, maximum, gamma=4.0):
     a = 1 + gamma * (mode - minimum) / scale
     b = 1 + gamma * (maximum - mode) / scale
 
-    return (a, b, loc, scale)
+    return a, b, loc, scale
+
+
+def _pert_fit_min_max_from_percentiles(low, mode, high, low_prec, high_prec, gamma):
+    """
+    Returns the maximum and the minimun of a PERT distribution with
+    percentiles corresponding to the inputs.
+
+    Examples
+    --------
+    >>> pert_fit_min_max_from_percentiles(2, 5, 7, 0.1, 0.9, 4)
+    (-1.2947082964619019, 8.747069754285215)
+    >>> pert_fit_min_max_from_percentiles(0, 5, 10, 0, 1, 4)
+    (0.0,10.0)
+    """
+
+    def equations(vars):
+        a, b = vars
+
+        # Big penalty for non-feasible solutions
+        if not (a < mode < b):
+            return [1e6, 1e6]
+
+        # Translate parameters to the beta distribution
+        alpha, beta, loc, scale = _pert_to_beta(a, mode, b, gamma)
+
+        # The CDF of pert is given by the CDF of beta with the substitution
+        # z = (x-min)/(max-min)
+        beta_low = (low - loc) / scale
+        beta_high = (high - loc) / scale
+
+        eq1 = sp.stats.beta.cdf(beta_low, alpha, beta) - low_prec
```

Comment:
perhaps add some gradient info here? there are many inputs for which the CDF evalutes to zero, some are arguably worse than others. e.g. Beta(2,2, x=0) and Beta(2,2, x=-100) both evaluate to 0.